roxctl/netpol/connectivity/map/map_test.go

```diff
@@ -211,9 +221,13 @@ func (d *connectivityMapTestSuite) TestAnalyzeNetpol() {
 				if tt.exposure {
 					expectedOutputFileName = "exposure_output." + formatSuffix
 				}
+				if tt.explain {
+					d.Equal(formatSuffix, defaultOutputFormat)
+					expectedOutputFileName = "explain_output." + formatSuffix
```

Comment:
```suggestion
					expectedOutputFileName = "explain_output.txt" 
```
only txt is supported for `explain`